picard/ui/options/scripting.py

```diff
@@ -232,6 +232,14 @@ def __init__(self, parent=None):
         self.last_selected_script_pos = 0
         self.ui.splitter.setStretchFactor(0, 1)
         self.ui.splitter.setStretchFactor(1, 2)
+        self.shortcut = QtWidgets.QShortcut(self.ui.script_list)
+        self.shortcut.setKey(QtGui.QKeySequence.Delete)
+        self.shortcut.activated.connect(self.processor_delete_item)
+
+    def processor_delete_item(self):
```

Comment:
Please conform to python method naming used in the rest of the project code.

Reply:
@sambhav ```script_delete_processor``` is ok?

Reply:
`delete_selected_script` should do.